Onboarding/Connector/Scripts/Documentation/BashScriptTsgAndFaq.md

```diff
@@ -0,0 +1,142 @@
+# For the bash script
+
+## FAQ
+
+### How do I run the script?
+
+Use the below commands
+
+For single subscription
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --subscription-id <subscription_id>
+```
+
+For new line separated subscription list in a file
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --file-path <file_path>
+```
+
+### Can I run this script for multiple subscriptions at once?
+
+Yes, use the `--file-path` option with a file containing newline-separated subscription IDs.
+
+### What permissions are needed to run this script?
+
+To ensure the onboarding script works seamlessly, the following permissions are required:
+
+1. **Contributor Permissions**: At the subscription level, this role is needed for executing steps related to:
+   1. Resource provider registration
+   2. Feature flag registration
+   3. Impact reporting connector resource creation
+
+2. **Role Creation Permissions**: To create a custom role that can read alerts generated from the subscription, one of the following roles is necessary:
+   1. User Access Administrator
+   2. Role Based Access Control Administrator
+
+In summary, a combination of these permissions will be sufficient to run the script effectively.
+
+### How do I enable debug mode?
+
+Uncomment the `set -x` at the beginning of the script to enable debug mode.
+
+### What should I do if I encounter a permission error?
+
+Verify your Azure role and permissions. You may need the help of your Azure administrator to grant you the necessary permissions or roles as defined the permissions section.
+
+### How can I verify if the connector is successfully created?
+
+Run the below command in AzCli
+
+```bash
+az rest --method get --url "https://management.azure.com/subscriptions/<subscription-id>/providers/Microsoft.Impact/connectors?api-version=2024-05-01-preview"
+```
+
+You should see a resource with the name: **AzureMonitorConnector**
+
+Or
+
+1. Navigate to [Azure Resource Graph Explorer](https://portal.azure.com/#view/HubsExtension/ArgQueryBlade)
+2. Run the below query
+
+   ```kql
+    impactreportresources
+        | where name == "AzureMonitorConnector" 
+            and type == "microsoft.impact/connectors"
+   ```
+
+3. The results should look like below, with a row for the connector resource
+
+![Viewing the Connector in ARG](Images/Viewing the Connector in ARG.png)
```

Comment:
Still can't see the image. '%20' should be added for the spaces, like the other links.

Reply:
Done